src/osx/cocoa/menu.mm

```diff
@@ -271,7 +271,7 @@ virtual void MacSetupHelpMenu()
                 }
             }
 
-            windowMenu = [[NSMenu alloc] initWithTitle:nsWindowMenuTitle];
+            windowMenu = [[NSMenu alloc] initWithTitle:nsAlternateWindowMenuTitle];
             NSMenuItem* windowMenuItem = [[NSMenuItem alloc] initWithTitle:nsWindowMenuTitle action:nil keyEquivalent:@""];
```

Comment:
Is it intentional that this doesn't use translated menu title? If so, it would certainly be helpful to have a comment explaining why, as it doesn't seem obvious to me...

Reply:
as you wrote later, the naming is not obvious, `nsAlternateWindowMenuTitle` is the translated one